.github/workflows/validate.yml

```diff
@@ -4,18 +4,20 @@ on:
   pull_request:
     branches:
     - main
+    - 2.x.x
 
 jobs:
   validate:
     runs-on: macOS-latest
+    timeout-minutes: 15
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       with:
         fetch-depth: 1
     - name: Install Dependencies
       run: |
         brew install mint
-        mint install NickLockwood/SwiftFormat@0.48.17 --no-link
+        mint install NickLockwood/SwiftFormat@0.51.15 --no-link
     - name: run script
       run: ./scripts/validate.sh
```

Comment:
SwiftFormat has a simple `lint` command which can get rid of that script for you.
Not sure about the year validation part of the script though.

Reply:
Yeah swiftformat can't verify the files have the correct header as far as I can tell. The script is a version of the one Apple uses